infogami/infobase/tests/conftest.py

```diff
@@ -1 +1,3 @@
+from __future__ import absolute_import
```

Comment:
ah, I actually deleted this file in my previous PR #57  and included a specific import in each of the tests that used `wildcard`

I'll review this PR further a bit later today, and I'll see if I can fix up and rebase myself since I caused some of the conflicts, and I have already looked at some of the issues with these imports as part of #58 (which I'll need to rebase too)